includes/event/event-capabilities.php

```diff
@@ -17,6 +17,7 @@ class Event_Capabilities {
 	private const TRASH          = 'trash_translation_event';
 	private const DELETE         = 'delete_translation_event';
 	private const EDIT_ATTENDEES = 'edit_translation_event_attendees';
+	private const EDIT_FORM_FIELD = 'edit_translation_event_form_field';
```

Comment:
We discussed adding new permissions per field, could you elaborate of the thinking to use `form_field` instead?
```suggestion
	private const EDIT_TITLE = 'edit_translation_event_title';
```

Reply:
Yes, on another thought, I was trying to be mindful of creating a lot of permissions and also considering the need to update this list of permissions when we add a new field to the form. So, I wanted to have a single permission that checks for the permission for any form field in this manner
```
current_user_can( 'edit_translation_event_form_field', $event->id(), 'event_description' )
```
So to check for the permission for any field we only need to pass the field name as an argument as shown above.

Reply:
I think in this case we could do something like:

```php
current_user_can( 'edit_translation_event', $event->id(), 'description' );
```

and modify the `edit_translation_event` capability so that it takes an optional field.

Reply:
However, I think it might be more explicit to have `edit_translation_event_description` instead. In practice there's not much difference between the two options, I personally don't have a strong preference for either.

Reply:
That's an interesting suggestion @psrpinto.